media/js/src/Asset.js

```diff
@@ -31,13 +28,12 @@ export default class Asset {
             const url = `https://vimeo.com/api/oembed.json?url=${vUrl}`;
             return fetch(url).then(function(response) {
                 if (response.status === 200) {
-                    return response.json();
+                    return response.json().then(function(d) {
```

Comment:
This works nicely. I'm not sure a `then` function is needed...could this be simplified?